docs/reference_guide.md

```diff
@@ -758,7 +758,7 @@ Constructors.
 
 ### 1. BPF
 
-Syntax: ```BPF({text=BPF_program | src_file=filename} [, usdt_contexts=[USDT_object, ...]])```
+Syntax: ```BPF({text=BPF_program | src_file=filename} [, usdt_contexts=[USDT_object, ...]] [, cflags=[arg1, ...]])```
 
```

Comment:
since you change it. How about make it complete, include `debug` and `hdr_file` as well?

Reply:
Happy to, if you can explain to me what they are.